src/interface.js

```diff
@@ -0,0 +1,54 @@
+$(document).ready( function() {
+  const game = new Game();
+  let rollNumber = 0;
+  let roll1;
+
+  $('button').on('click', function(e) {
+    roll = parseInt(e.target.innerHTML, 10);
+    rollNumber += 1;
+
+    //   to disable button  $("#btn").attr("disabled", true);
+    if ((rollNumber%2)==0) {
+      const frame = new Frame(roll1, roll);
+      try {
+        game.play(frame);
+        updateTable();
+      } catch (err) {
+        alert(err.message);
+      }
+      if (game.round < 10) {
+        rollNumber = 0;
+      } else if (game.round == 10) {
+        roll2 = roll;
+      }
+    } else if (rollNumber==3) {
+      roll3 = roll;
+    } else {
+      roll1 = roll;
+    }
+  });
+
+  function updateTable() {
+    const round = game.round;
+    let roll3 = 0;
+    if (round == 10) {
+      roll3 = game.frames[9].roll3;
+    }
+    $('#frame-table')
```

Comment:
Great idea/interesting you used a table to show the results. 
The interface is also linked very well to the game model